src/witan/models/load_data.clj

```diff
@@ -80,6 +80,21 @@
   {:column-names (apply-col-names-schema PopulationSchema csv-data)
    :columns (vec (apply-row-schema PopulationSchema csv-data))})
 
+(defmethod apply-rec-coercion :births
```

Comment:
I agree it can be left for now. Just one to think about from a design point of view.